### DISCUSSION TITLE
Future updates && support  2024

### DISCUSSION THREAD

**Discussion by YaremaVlad21** [react-native-voice/voice#511] [Q&A]
Hello `react-native-voice` team !

Is the any updates or support your library in nearest future?
Are you planned new releases ?
Are you planned support in future your library?

Thanks!

**Comment by brian-pinwheel** [comment]
I was debugging an issue I encountered when looking at the history of the file only to find the issue was fixed in a [PR ](https://github.com/react-native-voice/voice/pull/435)a year ago.  I immediately checked to make sure I had the latest version to find out there hadn't been a release in 2 years. 

Having maintained an open-source project myself in the past, I know it is a thankless job that takes a lot of time.  And I wanted to say thank you to ALL the contributors who have put their time into the project, but especially the maintainers.

My humble request is:
1. If you no longer have time to maintain the project, put a request for maintainers on the readme and see if others can help you out.
2. If you can find the time, it would be great to do a new release as other developers/committers who are using this project are submitting fixes and would love to be able to easily consume the fixes.

In the PR that fixed the issue, I found two reviewers (who I am guessing are maintainers) so tagging them here @jyant-labs @danieloi in case they have more information on the status of the project.

Again, thank you guys for all the work you have done!  You make open source work!

**Comment by sanduluca** [comment]
@safaiyeh  Can you get your attention here ?

**Comment by safaiyeh** [comment]
This project has been deprecated and archived. Please migrate to [`expo-speech-recognition`](https://github.com/jamsch/expo-speech-recognition). Thank you! ❤️
